### ISSUE TITLE
getNFTsByOwner not working

### ISSUE THREAD

**Issue by mprync** [primenumsdev/solana-nft-tools#4]
Hi, great wrapper, definitely helps simplify the mess of Solana's API.

So trying all of these methods seem to work, the example runs until I tried to use getNFTsByOwner.

I get a catch TypeError: Cannot read properties of undefined (reading 'value') exception. Am I doing something wrong?

The code I'm using is 100% the same code you have in your example on the Readme.

**Comment by primenumsdev** [comment]
Hi @InFaMoUsZero can you pls share an example of code you are running, so I can reproduce the issue.

**Comment by mprync** [comment]
```js
const NFTs = require('@primenums/solana-nft-tools');
const web3 = require('@solana/web3.js');

(async () => {
    // Create connection
    const conn = new web3.Connection(
        web3.clusterApiUrl('mainnet-beta'),
        'confirmed',
    );

    // Get all mint tokens (NFTs) from your wallet
    const walletAddr = 'YOUR_WALLET_ADDRESS_WITH_NFTS_INSIDE';

    // Additional function to get all NFTs from wallet address
    const allMyNFTs = await NFTs.getNFTsByOwner(conn, walletAddr);
    console.log('allMyNFTs', allMyNFTs);
})();
```

This returns the error I reported but if the wallet doesn't have any NFTs, it will return with an empty array, without an error.

It's also worth noting I am running this code in Quokka, not sure if that matters, I don't think it does.

**Comment by primenumsdev** [comment]
@InFaMoUsZero I have reproduced this error, it appears that Solana API was returning errors for the batch request where more than 1 item:
```
{
    jsonrpc: '2.0',
    error: {
      code: 429,
      message: 'Too many requests for a specific RPC call, contact your app developer or support@rpcpool.com.'
    },
    id: 'meta-ByQdFkC9cSMKTfsJoLp78SuGCiuU9esNitLP2CK1hFid'
  }
```

Which makes using batch requests useless, so I have updated implementation of `getNFTsByOwner ` to send separate request for every mint address.
And it works now.
I have published this update in a newer version: `0.2.1`.

**Comment by mprync** [comment]
Excellent mate, thanks a lot for taking the time to fix this!
